src/mods/settings.js

```diff
@@ -21,6 +21,7 @@ idrinth.settings = {
         alarmActive: false,
         bannedRaids: { },
         language: 'en',
```

Comment:
Changed this one as well: https://github.com/IDotD/Userscript/pull/237/commits/8274fa6b1df818280cfa892314d862ef068b1629

Maybe something got mixed up ?